kernel/packages/entryPoints/website.ts

```diff
@@ -91,15 +93,20 @@ namespace webApp {
 
   export async function loadUnity({ instancedJS }: InitializeUnityResult) {
     const i = (await instancedJS).unityInterface
+    let worldConfig: WorldConfig = globalThis.globalStore.getState().meta.config.world!
+
+    if (ENABLE_OLD_SETTINGS) {
+      worldConfig.enableOldSettings = ENABLE_OLD_SETTINGS
+    }
```

Comment:
same comment as before.. `globalThis.globalStore.getState().meta.config.world` might be undefined

Reply:
Done!